datumaro/components/annotation.py

```diff
@@ -787,3 +789,14 @@ class Caption(Annotation):
 
     _type = AnnotationType.caption
     caption: str = field(converter=str)
+
+
+@attrs(slots=True, order=False)
+class ImageResolution(Annotation):
+    """
+    Represents high and low resolution images.
+    """
+
+    _type = AnnotationType.image_resolution
+    hr_image = field()
+    upsampled_image: Optional[Image] = field(default=None)
```

Comment:
As I understand, in this task it is not an annotation. It's literally just the resized input image. I don't think storing it in annotations is the right way.

Reply:
Yes, you're right. Moved it to attributes.